contribs/gmf/apps/mobile/index.html

```diff
@@ -56,16 +56,16 @@
     </main>
     <nav class="nav-left" gmf-mobile-nav>
       <header>
-        <a class="go-back" href="#"></a>
+        <a class="go-back" href></a>
       </header>
       <!-- main menu -->
       <div class="active slide">
         <ul>
           <li>
-            <a href data-target="#background" data-toggle="slide-in" translate>Background</a>
+            <a href data-target="#background" data-toggle="slide-in">{{'Background' | translate}}</a>
```

Comment:
This change is necessary to (quick) fix the "invisible" submenu effect. But I'm not sur that we can still collect these strings to translate them...